src/platform/site-wide/sass/shame.scss

```diff
@@ -527,3 +527,7 @@ img[data-srcset] {
   list-style: none !important;
 }
 
+.circular-profile-image {
+  border-radius: 250px;
```

Comment:
did `border-radius: 50%` not work?

Reply:
Lol - I had no idea until you suggested, and yes, that does work, @ahay-agile6